### PR TITLE
diann added to containers

### PR DESCRIPTION
**Pull request by ypriverol**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file
  - [x] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

**Comment by ypriverol** [comment]
No bio.tools identifier has been defined.

File: diann/1.8.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y git g++ build-essential cmake zlib1g-dev libbz2-dev libboost-all-dev wget
```
Comment (osallou) [inline diann/1.8.0/Dockerfile]: why installing all of this, will hightly increase image size and not needed as installing a deb

File: diann/1.8.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Temp directory
+RUN mkdir /opt/software
+
+## Download everything
+RUN cd /opt/software && wget https://github.com/vdemichev/DiaNN/releases/download/1.8/diann_1.8.deb && dpkg -i diann_1.8.deb && rm -f diann_1.8.deb
```
Comment (osallou) [inline diann/1.8.0/Dockerfile]: a ... && apt-get install -y ./diann_1.8.deb should ok to install deb and related dependencies

File: diann/1.8.0/Dockerfile (hunk 1 above)
Comment (osallou) [reply diann/1.8.0/Dockerfile]: furthermore many are -dev , so include extras only needed for compilation...

**Pull request merged**